src/Account/Account.java

```diff
@@ -0,0 +1,73 @@
+package Account;
+
+public class Account {
+    public String id;
+    public String name;
+    public int balance;
+
+    public Account(String id, String name, int balance) {
+        this.id = id;
+        this.name = name;
+        this.balance = balance;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public int getBalance() {
+        return balance;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public void setBalance(int balance) {
+        this.balance = balance;
+    }
+
+    public void credit(int amount) {
+        if (amount > 0) {
+            balance = balance + amount;
+            System.out.println("Nap tien vao tai khoan thanh cong, balance = " + balance);
+        }
+        return;
```

Comment:
chỗ này  ko cần return đâu chị, return trong if hoặc trước if ấy
```suggestion
        if (amount > 0) {
            balance = balance + amount;
            System.out.println("Nap tien vao tai khoan thanh cong, balance = " + balance);
        }
```
hoặc
```suggestion
        if (amount < 1) {
        
           return;
        }
        
        balance = balance + amount;
        System.out.println("Nap tien vao tai khoan thanh cong, balance = " + balance);
```